docs/sections/header.html

```diff
@@ -0,0 +1,217 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+        <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script type="text/javascript" src="/cocoandeve-styleguides/js/bootstrap.js"></script>
+        <script type="text/javascript" src="/cocoandeve-styleguides/js/main.js"></script>
+	</head>
+	<body>
+        <header class="navbar navbar-expand-lg p-0 flex-wrap">
+            <div class="announcement-bar bg-primary-light text-center w-100">
+                <div class="container justify-content-center">
+                    <a href="#"><strong>Up to 25% off + Free Gift worth $25.40</strong></a>
```

Comment:
in css you add styles for the color of the font here, you can also add the font-style without using STRONG tag. Or better Bootstrap has an utility class .font-weight-bold